converter_test.go

```diff
@@ -828,10 +829,15 @@ func TestArrowToRecord(t *testing.T) {
 	var valids []bool           // AppendValues() with an empty valid array adds every value by default
 
 	localTime := time.Date(2019, 2, 6, 14, 17, 31, 123456789, time.FixedZone("-08:00", -8*3600))
+	localTimeFarIntoFuture := time.Date(9000, 2, 6, 14, 17, 31, 123456789, time.FixedZone("-08:00", -8*3600))
 
-	field1 := arrow.Field{Name: "epoch", Type: &arrow.Int64Type{}}
-	field2 := arrow.Field{Name: "timezone", Type: &arrow.Int32Type{}}
-	tzStruct := arrow.StructOf(field1, field2)
+	epochField := arrow.Field{Name: "epoch", Type: &arrow.Int64Type{}}
+	timezoneField := arrow.Field{Name: "timezone", Type: &arrow.Int32Type{}}
+	fractionField := arrow.Field{Name: "fraction", Type: &arrow.Int32Type{}}
+	timestampTzStruct2 := arrow.StructOf(epochField, timezoneField)
+	timestampTzStruct3 := arrow.StructOf(epochField, fractionField, timezoneField)
+	timestampNtzStruct := arrow.StructOf(epochField, fractionField)
+	timestampLtzStruct := arrow.StructOf(epochField, fractionField)
```

Comment:
We don't need fraction here?

Reply:
No. For LTZ we use local timezone (or rather session's TIMEZONE).